pipelines/operator-hosted-pipeline.yml

```diff
@@ -0,0 +1,255 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: operator-hosted-pipeline
+spec:
+  params:
+    - name: git_pr_branch
+    - name: git_pr_title
+    - name: git_pr_url
+    - name: git_repo_url
+    - name: git_username
+    - name: pyxis_url
+    - name: bundle_path
+    - name: preflight_latest_version
+    - name: ci_latest_version
```

Comment:
Shouldn't 'latest' really be 'min' instead? These will be used to verify that the testing was performed using a supported set of tools, right?